README.md

```diff
@@ -34,4 +34,11 @@ Instructions
 
 Deploy Requirements/Process
 --------------------------------------
-[TODO]
+Once you pull the file run the following commands in the main directory of the project
+```
+docker build .
```

Comment:
Maybe instead of saying below to grep through a (potentially long) list of image IDs, you could offer the alternative ```
`docker build -t <image> .`